script/release-notes/notes.js

```diff
@@ -81,10 +81,12 @@ const getNoteFromBody = body => {
   }
 
   const NOTE_PREFIX = 'Notes: '
+  const NOTE_HEADER = '#### Release Notes'
 
   let note = body
     .split(/\r?\n\r?\n/) // split into paragraphs
     .map(paragraph => paragraph.trim())
+    .map(paragraph => paragraph.startsWith(NOTE_HEADER) ? paragraph.slice(NOTE_HEADER.length).trim() : paragraph)
     .find(paragraph => paragraph.startsWith(NOTE_PREFIX))
```

Comment:
I'm new to this code so forgive me if I've misunderstood, but shouldn't we be taking the release notes from Clerk's comment, rather than from the PR body?

Reply:
No, we take them from the PR body at present!

Reply:
Fixed in 4e12b2c824e5a9a4ea9180cdffc02bf79fc36e12

Reply:
I was going to do it in a followup since it's a little more intrusive than the other fixes, but I guess it doesn't hurt to do it here.